package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
+    "build": "webpack -w --config webpack.config.js"
```

Comment:
if your config file name matches the standard naming convention, you don't need to specify it. `webpack -w` should be enough.

you may also want to move the watch flag into the webpack config rather than in this script so you can set it to true or false depending on if this is development vs. production

Reply:
@copelka so in package.json it should just be "build": "webpack -config" and then in webpack.config.js we just add watch: true ?